stack.py

```diff
@@ -102,5 +115,22 @@ def eval_program(program):
     return stack
 
 
+def try_float(string):
```

Comment:
Do we support any arguments that are not floats, currently? If no, I recommend removing this so that non-float arguments are an error sooner.